lib/sw-movies.js

```diff
@@ -0,0 +1,65 @@
+var express = require('express');
+var bodyParser = require('body-parser');
+var cors = require('cors');
+var swapi = require('swapi-node');
+
+
+function swRoute() {
```

Comment:
Same as above. I don't think you need the function here. You can just export the router directly. Makes for cleaner code.